diff --git a/src/hydra/material.cpp b/src/hydra/material.cpp
index 973e26f..bf6c497 100644
--- a/src/hydra/material.cpp
+++ b/src/hydra/material.cpp
@@ -130,8 +130,7 @@ class UsdToCycles {
       "principled_bsdf",
       {
           {TfToken("diffuseColor"), ustring("base_color")},
-          {TfToken("emissiveColor"), ustring("emission")},
-          {TfToken("specularColor"), ustring("specular")},
+          {TfToken("emissiveColor"), ustring("emission_color")},
           {TfToken("clearcoatRoughness"), ustring("coat_roughness")},
           {TfToken("opacity"), ustring("alpha")},
           // opacityThreshold
