diff --git a/moonray/CMakeLists.txt b/moonray/CMakeLists.txt
index 38175e1..7f08f51 100644
--- a/moonray/CMakeLists.txt
+++ b/moonray/CMakeLists.txt
@@ -12,10 +12,10 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/moonshine_dwa)
 endif()
 add_subdirectory(moonshine_usd)
 add_subdirectory(render_profile_viewer)
+add_subdirectory(scene_rdl2)
 if (BUILD_MATERIALX_SHADERS)
     add_subdirectory(materialx_shaders)
 endif()
-add_subdirectory(scene_rdl2)
 
 if (${BUILD_QT_APPS})
     add_subdirectory(moonray_gui)
