diff --git a/src/hydra/material.cpp b/src/hydra/material.cpp
index f4e85f0e..d3e6f8c5 100644
--- a/src/hydra/material.cpp
+++ b/src/hydra/material.cpp
@@ -228,5 +228 @@ void HdCyclesMaterial::Sync(HdSceneDelegate *sceneDelegate,
-  Initialize(renderParam);
-
-  const SceneLock lock(renderParam);
-
   const bool dirtyParams = (*dirtyBits & DirtyBits::DirtyParams);
@@ -248,2 +244,4 @@ void HdCyclesMaterial::Sync(HdSceneDelegate *sceneDelegate,
     if (network) {
+      Initialize(renderParam);
+      const SceneLock lock(renderParam);
       if (!_nodes.empty() && !dirtyResource) {
@@ -255,5 +253,9 @@ void HdCyclesMaterial::Sync(HdSceneDelegate *sceneDelegate,
       }
+
+      if (_shader->is_modified()) {
+        _shader->tag_update(lock.scene);
+      }
     }
     else {
-      TF_RUNTIME_ERROR("Could not get a material network for %s.", id.GetText());
+      TF_WARN("Could not get a material network for %s.", id.GetText());
     }
@@ -262,6 +264 @@ void HdCyclesMaterial::Sync(HdSceneDelegate *sceneDelegate,
-
-  if (_shader->is_modified()) {
-    _shader->tag_update(lock.scene);
-  }
-
   *dirtyBits = DirtyBits::Clean;
